### PR TITLE
add-apt-respository/adduser/alpine:add chinese translation

### PR DESCRIPTION
**Pull request by HugueLiu**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by agnivade** [comment]
If we know someone, then sure. Otherwise I don't think we can do much. I have run a couple of them under google translate just as a smoke test.

File: pages.cn/linux/apk.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# apk
+
+> Alpine linux的包管理工具.
+
+- 从所有的远程仓库中更新仓库索引
```
Comment (sbrl) [inline pages.cn/linux/apk.md]: I believe a colon is needed at the end of this line, at least in English. Is this not the case in Chinese?

Comment (HugueLiu) [reply pages.cn/linux/apk.md]: yes, there should be a colon, it's my fault. I have corrected the error

**Comment by waldyrious** [comment]
@HugueLiu shouldn't the tokens (`{{...}}`) be translated as well?

**Comment by HugueLiu** [comment]
> @HugueLiu shouldn't the tokens (`{{...}}`) be translated as well?

I have though about that, but I'm not sure whether the contents in {{}} should be translated. If should, I will translate them.

**Comment by agnivade** [comment]
Yes, please go ahead and translate them.

**Pull request merged**